modules/dailymotionBidAdapter.md

```diff
@@ -9,10 +9,11 @@ Maintainer: ad-leo-engineering@dailymotion.com
 # Description
 
 Dailymotion prebid adapter.
+Supports video ad units in instream context.
 
 # Configuration options
 
-Before calling this adapter, you need to set at least the API key in the bid parameters:
+Before calling this adapter, you need at least set an video adUnit in an instream context and the API key in the bid parameters:
```

Comment:
Before calling this adapter, you need **to at least set a** video adUnit in an instream context and the API key in the bid parameters: